lib/elixir/lib/protocol.ex

```diff
@@ -677,6 +680,13 @@ defmodule Protocol do
     ] ++ new_signatures
   end
 
+  defp get_protocol_functions({_name, _kind, _meta, clauses}) do
+    Enum.find_value(clauses, fn
+      {_meta, [:functions], [], clauses} -> clauses
+      _ -> nil
+    end)
```

Comment:
```suggestion
    end) || raise "could not find protocol functions"
```

Reply:
I really missed `Enum.find_value!` when I was writing this one, I actually double checked if we didn't add it 😄